syntax = "proto3";

option go_package = "pb/game";

message BuffContextPBComp
{
  float damage_value = 1;
}

message BuffPBComp
{
  uint64 buff_id = 1;
  uint32 buff_table_id = 2;
  uint32 ability_id = 3;
  uint64 parent_entity = 4;
  uint32 layer = 5;
  uint64 caster = 6;
  bool triggerdamage = 7;
}
